addons/l10n_se/data/account.account.template.csv

```diff
@@ -125,7 +121,6 @@
 "a2890","2890","Övriga kortfristiga skulder","account.data_account_type_current_liabilities","l10nse_chart_template","False"
 "a2910","2910","Upplupna löner","account.data_account_type_current_liabilities","l10nse_chart_template","False"
 "a2920","2920","Upplupna semesterlöner","account.data_account_type_current_liabilities","l10nse_chart_template","False"
-"a2930","2930","Upplupna pensionskostnader","account.data_account_type_current_liabilities","l10nse_chart_template","False"
```

Comment:
This is not listed as the recommended minimum. However, it's included in the complete BAS 2020 and should not be removed.

Reply:
Its not in minimum and should not be included.
There are may accounts in extended

Reply:
Not in minimum ( https://github.com/odoo/odoo/pull/53092#discussion_r454259845 )